src/Symfony/Bundle/FrameworkBundle/Resources/config/web.php

```diff
@@ -145,5 +147,11 @@
         ->set('controller.cache_attribute_listener', CacheAttributeListener::class)
             ->tag('kernel.event_subscriber')
 
+        ->set('controller.expression_language', ExpressionLanguage::class)
+            ->args([service('cache.request_payload_value_resolver_expression_language')->nullOnInvalid()])
+
+        ->set('cache.request_payload_value_resolver_expression_language')
```

Comment:
either the EL service is named like proposed and this needs to ba aligned, or we change the name of the EL service, but the current mix looks inconsistent to me
```suggestion
        ->set('cache.controller_expression_language')
```

Reply:
@nicolas-grekas true. i prefer suggested change.
fixed https://github.com/symfony/symfony/pull/58273/commits/b630a67b349d0f61ad8a89353b3efbfb24374d6c